src/stories/components/card-info-member/card-info-member.tsx

```diff
@@ -61,15 +61,15 @@ const CardInfoMember = ({ contributorCommitment }: Props) => {
                         overflow: 'hidden',
                         width: '207px',
                       }}>
-                      {contributor.email}
+                      {contributor.email.toLowerCase()}
                     </TypographyEmail>
                   </CustomPopover>
                   : <TypographyEmail isLight={isLight}
                     style={{
                       textOverflow: 'ellipsis',
                       overflow: 'hidden',
                     }}>
-                    {contributor.email}
+                    {contributor.email && contributor.email.toLowerCase()}
```

Comment:
Maybe using the optional chaining operator it is easy to read and code `contributor?.email?.toLowerCase()`. Please, note that `contributor.email` is used before, if it could be undefined | null, should make sense to use the optional chaining the two times it is used before

Reply:
done